### PR TITLE
Create python-package.yml

### LINKED ISSUE #36: CI is not working
We used to use Travis CI, but it is no longer permitted for Google projects.

If I understand correctly, we can use Github Actions. Someone needs to port `.travis.yml` to whatever format Github Actions requires.

### PR DESCRIPTION
**Pull request by bashi**
Fix #36.

### CODE REVIEW

**Comment by bashi** [comment]
@ricea PTAL

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    branches: [ "master" ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ["3.9", "3.10", "3.11"]
```
Comment (ricea) [inline .github/workflows/python-package.yml:16]: Are older versions not supported by Github Actions? Previously we were testing on Python 2 too.

**Review by ricea — APPROVED** [review]
lgtm

File: .github/workflows/python-package.yml (hunk 1 above)
Comment (bashi) [reply .github/workflows/python-package.yml:16]: Looks like https://github.com/actions/python-versions does not support Python 2. I picked three versions to reduce tasks.

Comment (ricea) [reply .github/workflows/python-package.yml:16]: It looks like 3.5 is the oldest version we can test with. Let's assume that it isn't broken in intervening versions. Also, we can use 3.x to automatically pick up new stable versions so we will notice if a new version is broken.
```suggestion
        python-version: ["3.5", "3.10", "3.x"]
```

Comment (bashi) [reply .github/workflows/python-package.yml]: Python 3.7 and lower reached end-of-life. Tests also fail on the latest version (3.12). We should fix tests but it should be done in a separate PR?

Comment (ricea) [reply .github/workflows/python-package.yml]: For the time being we also support Python 2, which is also end-of-life, so our support decisions are not necessarily tied to Python's.

Let's stick with this for now and fix 3.12 in a separate PR.

Comment (bashi) [reply .github/workflows/python-package.yml]: Makes sense. Replaced 3.8 with 3.5. Can you squash & merge? It seems that I don't have the permission.

Comment (bashi) [reply .github/workflows/python-package.yml]: Hmm 3.5 didn't work. Changed to 3.7.

**Pull request merged**